docs/_styles/extra.css

```diff
@@ -20,11 +20,6 @@ table {
     color: var(--md-accent-fg-color);
 }
 
-.celltag_remove_input,
-.celltag_remove_output,
-.celltag_hide_output,
-.celltag_Remove_input,
-.celltag_Remove_all_output,
-.celltag_Remove_single_output {
+.celltag_remove_output .output_wrapper {
```

Comment:
Yes, but it's still a hack. According to the docs, the cell content shouldn't even be rendered into the HTML.

The fact that this feature doesn't work in both `mknotebooks` and `mkdocs-jupyter` (which I played around with briefly yesterday) makes me believe that it might actually a problem in `nbconvert`, which both libraries invoke to do the actual conversion of the notebook. In the meantime, I've reported the behavior here, in case it actually is caused by mknotebooks itself: https://github.com/greenape/mknotebooks/issues/1117